lib/hosts.nix

```diff
@@ -3,7 +3,7 @@
   nixpkgs,
   ...
 }: rec {
-  mkHost = base: system: extraModules:
+  mkHost = base: system: extraModules: domain: hostname:
```

Comment:
nit;
```suggestion
  mkHost = base: system: extraModules: domain: hostName:
```

for consistency with the option name

Reply:
Sure.